samples/sid_end_device/CMakeLists.txt

```diff
@@ -46,6 +46,8 @@ if(CONFIG_SID_END_DEVICE_CLI)
     target_sources(app PRIVATE
         src/cli/app_dut.c
         src/cli/app_shell.c
+        src/cli/sbdt_shell.c
```

Comment:
Shouldt sdbt be added conditionally when the file transfer config is enable?

Reply:
ok